### PR TITLE
Some Fixes

### PR DESCRIPTION
**Pull request by IParallel**
I noticed that entity positions where abnormal and i found out the problem was a typo where we were setting the position to the actual delta change

### CODE REVIEW

File: MineSharp.Bot/Plugins/WindowPlugin.cs [hunk 1]
```diff
@@ -450,7 +486,10 @@ private Task HandleOpenWindow(OpenWindowPacket packet)
 
         windowInfo = windowInfo with { Title = packet.WindowTitle };
         Logger.Debug($"Received Open Window Packet id={packet.WindowId}");
-        OpenWindow(packet.WindowId, windowInfo);
+        Task.Delay(100).ContinueWith(t =>
```
Comment (psu-de) [inline MineSharp.Bot/Plugins/WindowPlugin.cs:489]: Is there a reason why we wait 100ms before opening the window?

Comment (IParallel) [reply MineSharp.Bot/Plugins/WindowPlugin.cs:489]: i think i sent a video on the discord addressing that temporal change you may want to watch it, i still didn't found the reason of why the task never executes and always times out, but adding a delay make it works.

Comment (psu-de) [reply MineSharp.Bot/Plugins/WindowPlugin.cs:489]: hm seems to be a race condition. PR #71 addresses a lot of bugs regarding race conditions, maybe it fixes it.

**Comment by Klotzi111** [comment]
With entity position there was also a problem where the SetVelocityPacket set the velocity of the entity as their position and thus entities often have almost zero coordinates.
I fixed those problems in #71. You should try it

**Pull request closed without merging**